src/calculator-ui-class.js

```diff
@@ -145,10 +146,10 @@ export class CalculatorUI {
         result = new CubeCommand(this.calculator, current);
         break;
       case VALUES.TENTOTHEX:
-        result = new TenToTheXCommand(this.calculator, current);
+        result = new PowerOfTenCommand(this.calculator, current);
```

Comment:
previous change seems to be the correct one, isn't it?

case VALUES.TENTOTHEX:
        result = new TenToTheXCommand(this.calculator, current);